### ISSUE TITLE
Return null if IsNull property is true, but there is a value.

### ISSUE THREAD

**Issue by Alfetta159** [mmkiwi/MdbReader#5]
In MdbDataRow.cs, at line: 165, why does it return `null` when `IsNull` is `true`? Does `IsNull` have something to do with the nullability of the column? I am getting nulls back when it clearly shows a non-null `Value`. This happens with string values. 

```
    /// <summary>
    /// Gets the value of the specified column in its native format.
    /// </summary>
    /// <param name="columnName">The column name</param>
    /// <returns>The underlying data object, or null if the field is null.</returns>
    /// <throws cref="ArgumentNullException"><c>columnName</c> is <c>null</c>.</throws>
    /// <throws cref="IndexOutOfRangeException"> No column with the specified name was found.</throws>
    public object? GetValue(string columnName)
    {
        var fieldValue = GetFieldValue(columnName);
        if (fieldValue.IsNull) return null;
        return fieldValue.Value;
    }
```

**Comment by mmkiwi** [comment]
Sorry it's taken so long to get to this, I can take a look and see if IsNull is getting set too quickly. The IsNull comes from the Null Mask (see https://github.com/mdbtools/mdbtools/blob/dev/HACKING.md),  I wonder if for strings, we should be ignoring that mask and just pulling the string value. What's the max length of the string fields you're seeing this with?

**Comment by Alfetta159** [comment]
I haven't had time to delve into this either. The version that I forked is working well enough for us.

tl;dr

The .mdb file that we have has five tables, only one of which we really care about and so I just forked it, (which I hate doing BTW) so it hasn't been too much of a problem. I hope to have a little time to look at it in the future (famous last words.)
